apps/web/components/booking/BookingListItem.tsx

```diff
@@ -275,6 +275,56 @@ function BookingListItem(booking: BookingItemProps) {
   const showRecordingsButtons =
     (booking.location === "integrations:daily" || booking?.location?.trim() === "") && isPast && isConfirmed;
 
+  const meetingLink = (link: string) => {
+    let linkOrType = link;
+    if (link.startsWith("https://meet.google")) {
+      linkOrType = "integrations:google:meet";
+    }
+    if (link.startsWith("https://zoom")) {
+      linkOrType = "integrations:zoom";
+    }
+    const app = guessEventLocationType(linkOrType);
+    if (app) {
+      if (link.startsWith("https")) {
+        return (
+          <div className="flex">
+            <Link
+              href={link}
+              onClick={(e) => e.stopPropagation()}
+              className="flex items-center gap-1 text-sm leading-6 text-blue-400 hover:underline">
+              <img src={app.iconUrl} alt={app.label} height={16} width={16} />
+              Join {app.label}
+            </Link>
+          </div>
+        );
+      }
+      const url = booking.references[0].meetingUrl;
```

Comment:
Is this the correct way to get urls from integrations @PeerRich ?

Reply:
i believe so. asking @hariombalhara to review

Reply:
https://github.com/calcom/cal.com/pull/11866 this looks better approach

Reply:
I didn't know you wanted Join Meeting link to go booking page